### PR TITLE
Task completed

### PR DESCRIPTION
**Pull request by Vedroid**

### CODE REVIEW

File: src/main/java/core/basesyntax/ApplicationStarter.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import core.basesyntax.controller.ConsoleHandler;
+import core.basesyntax.db.Storage;
+
+public class ApplicationStarter {
+    public static void main(String[] args) {
+        ConsoleHandler consoleHandler = new ConsoleHandler();
+        consoleHandler.handle();
+
+        Storage.bets.forEach(bet ->
+                System.out.println("Value: " + bet.getValue() + ", risk: " + bet.getRisk()));
```
Comment (Max-Pochepets) [inline src/main/java/core/basesyntax/ApplicationStarter.java:12]: If you have multiple db's, let's output them all

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    UserDao userDao = new UserDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        enterUserName(scanner);
+        enterBet(scanner);
+        scanner.close();
+    }
+
+    private void enterUserName(Scanner scanner) {
```
Comment (Max-Pochepets) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:23]: what if name will contain smth else then letters?

**Review by Max-Pochepets — APPROVED** [review]
:+1:

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+    UserDao userDao = new UserDaoImpl();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: What's missing?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+    private void enterUserName(Scanner scanner) {
+        while (true) {
+            System.out.println("Enter your name:");
+            String name = scanner.nextLine();
+            if (!name.isEmpty() && name.matches("[A-Za-z]+")) {
+                userDao.add(new User(name));
+                return;
+            }
+            System.out.println("Please enter correct data.");
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let's add an option to exit the application

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        while (true) {
+            System.out.println("Enter value and risk for your bet:  (enter 'q' to exit)");
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:48]: let's not create Bet if the input was empty

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 5 above)
Comment (Vedroid) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java:48]: if the input was empty Bet will not be created
we will not get to the line where Bet is created

File: src/main/java/core/basesyntax/ApplicationStarter.java (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/ApplicationStarter.java:12]: it would be easier to override toString() for Bet

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**